tests/linalg_test.py

```diff
@@ -534,6 +534,53 @@ def testQrBatching(self, shape, dtype, rng_factory):
     qs, rs = vmap(jsp.linalg.qr)(args)
     self.assertTrue(onp.all(onp.linalg.norm(args - onp.matmul(qs, rs)) < 1e-3))
 
+  @parameterized.named_parameters(jtu.cases_from_list(
+      {"testcase_name":
+       "_shape={}".format(jtu.format_shape_dtype_string(shape, dtype)),
+       "shape": shape, "dtype": dtype, "rng_factory": rng_factory}
+      for shape in [(1, 1), (4, 4), (2, 3, 5), (5, 5, 5), (20, 20), (5, 10)]
+      for dtype in float_types + complex_types
+      for rng_factory in [jtu.rand_default]))
+  def testCond(self, shape, dtype, rng_factory):
+    _skip_if_unsupported_type(dtype)
+    rng = rng_factory()
+    args_gen = lambda p: lambda: [rng(shape, dtype), p]
+    for pnorm in ['fro', np.inf, -np.inf, 1, -1, 2, -2]:
+      args_maker = args_gen(pnorm)
+      if pnorm not in [2, -2] and len(set(shape[-2:])) != 1:
+        with self.assertRaises(onp.linalg.LinAlgError):
+          np.linalg.cond(*args_maker())
+      else:
+        self._CheckAgainstNumpy(onp.linalg.cond, np.linalg.cond, args_maker,
+                                check_dtypes=False, tol=1e-3)
+
```

Comment:
Can you add a call to `self._CompileAndCheck` here? That should hopefully catch `jit` issues.

Reply:
I've added it but commented it out for now, as I'm getting this error:

```
ERROR: testCond_shape=float64[4,4] (linalg_test.NumpyLinalgTest)
testCond_shape=float64[4,4] (linalg_test.NumpyLinalgTest)
testCond_shape=float64[4,4](shape=(4, 4), dtype=<class 'numpy.float64'>, rng_factory=<function rand_default at 0x1279d1c20>)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/usr/local/lib/python3.7/site-packages/absl/testing/parameterized.py", line 263, in bound_param_test
    test_method(self, **testcase_params)
  File "/Users/shogg/git/jax/tests/linalg_test.py", line 566, in testCond
    check_dtypes=False, rtol=1e-03, atol=1e-03)
  File "/Users/shogg/git/jax/jax/test_util.py", line 724, in _CompileAndCheck
    monitored_ans = cfun(*args)
  File "/Users/shogg/git/jax/jax/api.py", line 150, in f_jitted
    name=flat_fun.__name__)
  File "/Users/shogg/git/jax/jax/core.py", line 605, in call_bind
    outs = primitive.impl(f, *args, **params)
  File "/Users/shogg/git/jax/jax/interpreters/xla.py", line 449, in _xla_call_impl
    compiled_fun = _xla_callable(fun, device, backend, name, *map(arg_spec, args))
  File "/Users/shogg/git/jax/jax/linear_util.py", line 223, in memoized_fun
    ans = call(fun, *args)
  File "/Users/shogg/git/jax/jax/interpreters/xla.py", line 466, in _xla_callable
    jaxpr, (pvals, consts, env) = pe.trace_to_subjaxpr(fun, master, False).call_wrapped(pvals)
  File "/Users/shogg/git/jax/jax/linear_util.py", line 152, in call_wrapped
    ans = self.f(*args, **dict(self.params, **kwargs))
  File "/Users/shogg/git/jax/jax/test_util.py", line 706, in wrapped_fun
    return fun(*args)
  File "/Users/shogg/git/jax/tests/linalg_test.py", line 564, in <lambda>
    self._CompileAndCheck(lambda x: np.linalg.cond(x, pnorm),
  File "/Users/shogg/git/jax/jax/third_party/numpy/linalg.py", line 57, in cond
    if nan_mask.any():
  File "/Users/shogg/git/jax/jax/core.py", line 349, in __bool__
    def __bool__(self): return self.aval._bool(self)
  File "/Users/shogg/git/jax/jax/abstract_arrays.py", line 37, in error
    raise TypeError(concretization_err_msg(fun))
TypeError: Abstract value passed to `bool`, which requires a concrete value. The function to be transformed can't be traced at the required level of abstraction. If using `jit`, try using `static_argnums` or applying `jit` to smaller subfunctions instead.
```

Basically, the function signature of the original `np.linalg.cond` can [sometimes take a string for one of the parameters](https://docs.scipy.org/doc/numpy/reference/generated/numpy.linalg.cond.html), which would interfere with `jit`. Not sure how to proceed here, any advice?

Reply:
Worth adding, when I run this test in a terminal but use static args on all the parameters in `cond`, things seem fine.

Reply:
Appear to be getting the same issue with the implementation of `tensorinv`